api/DocumentTimeline.json

```diff
@@ -0,0 +1,101 @@
+{
+  "api": {
+    "DocumentTimeline": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/DocumentTimeline",
+        "support": {
```

Comment:
Please order the entries in this order:
"webview_android", "chrome", "chrome_android", "edge", "edge_mobile", "firefox", "firefox_android", "ie", "opera", "opera_android".

Thank you!

Reply:
Order changed. May I suggest that the required order be tested for?

Reply:
Yes, we should add a test, unfortunately, it is not trivial (read: not possible in the schema), and people usually do the error once then copy/paste another structure, so it does happen only once per contributor ;-) But PRs welcome of course.